README.md

```diff
@@ -2,4 +2,36 @@
 
 This is a website on the world-wide web utilizing hyper-links to assist you navigating the website of the Protocol Berg conference in Berlin.
 
-* <https://protocol.berlin>
+- <https://protocol.berlin>
+
+## Install
+
+Minimum required Node.js version is 18.
+
+```
+npm install
+```
+
+## Develop
+
+```
+npx gatsby develop
+```
+
+## Build
+
+A Pretalx token is required to pull the speakers from [speak.ticketh.xyz](https://speak.ticketh.xyz/protocol-berg/).
+
+Go to you Account Settings and grab the token from the bottom of the page.
+
+Create a `.env.production` file with the following content:
+
+```
+PRETALX_TOKEN=your-token-here
```

Comment:
where is it actually used? I only see the stuff inlined here and no usage of PRETALX_TOKEN